src/ui/view/dataverse/component/filters/dateRangeFilter/dateRangeFilter.tsx

```diff
@@ -0,0 +1,61 @@
+import type { FC } from 'react'
+import { useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { DateRange } from '@/ui/component/dateRangePicker/dateRangePicker'
+import { DateRangePicker } from '@/ui/component/dateRangePicker/dateRangePicker'
+import { Filter } from '@/ui/view/dataverse/component/filters/filter'
+import './dateRangeFilter.scss'
+
+export type SelectionOption = {
+  selected: boolean
+  value: string
+  name: string
+  disabled: boolean
+}
+
+type FilterLabelProps = {
+  label: string
+}
+export const FilterLabel: FC<FilterLabelProps> = ({ label }) => (
```

Comment:
`FilterLabel` is defined 3 times and also exported 3 times 🤔

Reply:
Thanks for pointing this out! Duplicated types has been removed in [b1810dd](https://github.com/okp4/dataverse-portal/pull/473/commits/b1810dd1267b0ca44824dc3eb5101c43ae26249b).